docs/source/README.rst

```diff
@@ -1 +1,172 @@
-.. include:: ../../README.rst
+========
+Overview
+========
+
+
+
+| |Tests|_ |Documentation|_ |nix|_
+| |GitHub|_ |PyPI|_  |CondaForge|_ |Binder|_
+| |gitter|_ |OpenHub|_
+
+.. only:: latex
```

Comment:
I need to think about how to handle these `only:: latex` blocks. We do this because we have a custom [index.html](https://github.com/usnistgov/fipy/blob/master/docs/source/_templates/index.html). The README gets linked from the "About FiPy" link on the home page. At the time I did all this, I didn't care for the redundant text between those two pages, so excluded it in the HTML with `only:: latex`. The thing is, that text *should* show up when viewing the README in GitHub or `cat README.rst` from the command-line.

The right answer is probably to tolerate the redundancy.

Reply:
Yeah, this seemed a bit awkward and fiddly, just having the docs be the same makes sense to _me_. When in doubt, simplify, right?

Reply:
Agreed. FYI, I'm out for a bit, but will work on this part when I get back later next week.

Reply:
No more `.. only:: latex`